post_proc_test.py

```diff
@@ -75,6 +75,9 @@ def test_extrai_diarios(self):
                     for ato_esperado,ato_obtido in zip(diario_esperado.atos, diario_obtido.atos):
                         self.assertEqual(ato_esperado.cod, ato_obtido.cod, f'Caso: {case.desc}\nMunicípio: {diario_esperado.municipio}')
                         self.assertEqual(ato_esperado.nomeacoes > 0, ato_obtido.possui_nomeacoes, f'Caso: {case.desc}\nMunicípio: {diario_esperado.municipio}\nTexto:{ato_obtido.texto}')
+                        if (ato_esperado.nomeacoes > 0):
```

Comment:
Por isso que os testes passaram. Pois você só confere os CPFs quando o número de nomeações é maior que zero. Você deve conferir de todos. Quando não for nomeação é para retornar `cpf_nomeacao` nil ou vazio.